languages/typescript/src/main/kotlin/io/vrap/codegen/languages/typescript/server/ServerRenderer.kt

```diff
@@ -232,6 +240,10 @@ class ServerRenderer @Inject constructor(
                 val joiType = this.toJoiVrapType()
                 "import { ${joiType.simpleClassName} } from '${joiType.`package`}'"
             }
+            is VrapEnumType -> {
```

Comment:
This was missing and is required because some of our query parameters (e.g. `state`) use an enum type that are contained in a separate module.